lib/api/model/model.dart

```diff
@@ -569,7 +569,7 @@ sealed class Message {
   String content;
   final String contentType;
 
-  // final List<MessageEditHistory> editHistory; // TODO handle
+  final List<MessageEditHistory>? editHistory; // TODO handle
```

Comment:
This TODO comment should be removed — it's calling for us to do what this PR is doing :slightly_smiling_face: